docs/pages/access-controls/access-graph/entra-id.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Analyze Entra ID policies with Teleport Policy
+description: Describes how to import and visualize Entra ID policies using Teleport Policy and Access Graph.
+---
+
+The Microsoft Entra ID integration in Teleport Identity synchronizes your Entra ID directory into your Teleport cluster,
+and offers insights into relationships in your Entra ID directory.
+Additionally, when Entra ID is used as an SSO identity provider, Teleport Policy visualizes
+SSO grants across your services.
+
+<Notice type="note">
+SSO grant analysis is currently only supported in situations where Entra ID acts as the identity provider,
+and AWS accounts are set-up as relying parties using AWS IAM role federation.
+
+Support for additional relying parties will be added in the future.
+</Notice>
+
+Utilizing the Access Graph to analyze Entra ID policies necessitates the setup of the Access Graph service,
```

Comment:
I would move this paragraph into the Prerequisites section and restructure it, since I find the current wording a little awkward:

```markdown
You must have Access Graph enabled in your Teleport account.
```

The paragraphs that follow this one can also go under "Prerequisites".